ansible/roles/netbox/defaults/main.yml

```diff
@@ -1128,7 +1128,7 @@ netbox__nginx__dependent_servers:
         proxy_connect_timeout {{ netbox__app_timeout }};
         proxy_send_timeout {{ netbox__app_timeout }};
         proxy_read_timeout {{ netbox__app_timeout }};
-        add_header P3P 'CP="ALL DSP COR PSAa PSDa OUR NOR ONL UNI COM NAV"';
+        add_header P3P 'CP="ALL DSP COR PSAa PSDa OUR NOR ONLY UNI COM NAV"';
```

Comment:
I'm not sure about this change. I did a brief search for Compact Policies and found [this website](http://etutorials.org/Mobile+devices/mobile+location+services/Appendix+D.+Platform+for+Privacy+Preferences+P3P/4+Compact+Policies/) where `ONL` is used as a category:
```
"PHY" | ; for <physical/>
"ONL" | ; for <online/>
"UNI" | ; for <uniqueID/>
"PUR" | ; for <purchase/>
"FIN" | ; for <financial/>
"COM" | ; for <computer/>
"NAV" | ; for <navigation/>
"INT" | ; for <interactive/>
"DEM" | ; for <demographic/>
"CNT" | ; for <content/>
"STA" | ; for <state/>
"POL" | ; for <political/>
"HEA" | ; for <health/>
"PRE" | ; for <preference/>
"LOC" | ; for <location/>
"GOV" | ; for <government/>
"OTC"   ; for <other-category/>
```
Is there a way to mark this as a false positive?

Reply:
False positives (lines to ignore) can be added to the ./.codespell.exclude file at the root of the project.

Reply:
Thanks, I'll check it out before merging the PR.